README.md

```diff
@@ -104,6 +104,13 @@ Type: `boolean`. Default: `false`.
 By default, links pointing to other origins (protocol + domain) are not hijacked.
 If this option is `true`, these links will still be hijacked.
 
+##### skipFragment
+
+Type: `boolean`. Default: `false`.
+
+By default, links starting with fragments (e.g. `href="#foo"`) are not hijacked.
+If this option is `true`, these links will still be hijacked.
```

Comment:
I'm struggling with the name `skipFragment`, I think. I interpret it as, "I want the link hijacker module to skip the hijacking of fragment links, so I set `skipFragment` to `true`." I am having a hard time making sense of the opposite implication.

Reply:
Oh I think the default is actually `true`, right?

Reply:
Yes, the default should be `true`. I confused myself several times about this while creating the PR.

If we change the default value in the README, does this make sense

Reply:
Yep! It makes perfect sense if the README changes.

Reply:
Ok, pushed the change. *All* of the defaults in the README were reversed, so I fixed 'em.

Reply:
Oh good catch!